In [ ]:
!wget -O AI-111.xlsx "https://raw.githubusercontent.com/glgunderson/AI-Perceptions/main/data/AI-111.xlsx"

--2026-03-19 07:00:13--  https://raw.githubusercontent.com/glgunderson/AI-Perceptions/main/data/AI-111.xlsx
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.111.133, 185.199.110.133, 185.199.109.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.111.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 28471 (28K) [application/octet-stream]
Saving to: ‘AI-111.xlsx’

AI-111.xlsx         100%[===================>]  27.80K  --.-KB/s    in 0.001s  

2026-03-19 07:00:13 (41.2 MB/s) - ‘AI-111.xlsx’ saved [28471/28471]



In [ ]:
import pandas as pd
df = pd.read_excel("AI-111.xlsx")
df.head()

,Age,Education,Frequency,Job status,Field,Field_10_TEXT,Perceptions_1,Perceptions_2,Perceptions_3,Perceptions_4,...,Procedural Fairness_2,Outcome Fairness_1,Outcome Fairness_2,Outcome Fairness_3,Performance Pressure_1,Performance Pressure_2,Performance Pressure_3,Competency_1,Competency_2,Competency_3
0,29,1,5,5,1,NaN,5,4,4,4,...,4,1,5,1,4,4,4,2,5,5
1,20,1,2,1,"7,8,9",NaN,4,3,3,3,...,4,3,3,2,4,2,3,2,2,3
2,20,1,4,5,"1,2,3,4,9",NaN,4,4,4,4,...,3,2,2,3,4,4,4,4,4,4
3,23,3,5,1,"1,4",NaN,4,4,4,5,...,3,2,4,3,4,4,4,4,4,5
4,20,1,3,5,"1,2,3,4,9",NaN,4,4,4,4,...,4,4,4,4,4,4,2,4,4,4


In [ ]:
df.info()
df.describe()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 110 entries, 0 to 109
Data columns (total 49 columns):
 #   Column                                              Non-Null Count  Dtype 
---  ------                                              --------------  ----- 
 0   Age                                                 110 non-null    int64 
 1   Education                                           110 non-null    int64 
 2   Frequency                                           110 non-null    int64 
 3   Job status                                          110 non-null    int64 
 4   Field                                               110 non-null    object
 5   Field_10_TEXT                                       7 non-null      object
 6   Perceptions_1                                       110 non-null    int64 
 7   Perceptions_2                                       110 non-null    int64 
 8   Perceptions_3                                       110 non-null    int64 
 9   Perception

,Age,Education,Frequency,Job status,Perceptions_1,Perceptions_2,Perceptions_3,Perceptions_4,Perceptions_5,AI Expectations_1,...,Procedural Fairness_2,Outcome Fairness_1,Outcome Fairness_2,Outcome Fairness_3,Performance Pressure_1,Performance Pressure_2,Performance Pressure_3,Competency_1,Competency_2,Competency_3
count,110.000000,110.000000,110.000000,110.000000,110.000000,110.000000,110.000000,110.000000,110.000000,110.000000,...,110.000000,110.000000,110.000000,110.000000,110.000000,110.000000,110.000000,110.000000,110.000000,110.000000
mean,22.527273,1.372727,3.927273,2.536364,3.890909,3.754545,3.618182,4.172727,4.463636,4.081818,...,3.554545,2.572727,3.890909,2.454545,3.854545,3.872727,3.836364,3.909091,3.809091,3.881818
std,3.161434,0.647636,1.037897,1.769659,0.860397,0.869318,0.928467,0.822383,0.569521,0.791373,...,0.711458,1.035685,0.870995,1.001250,0.833214,0.847112,0.923965,0.883544,0.990741,0.786723
min,19.000000,1.000000,1.000000,1.000000,1.000000,1.000000,1.000000,1.000000,3.000000,2.000000,...,1.000000,1.000000,2.000000,1.000000,1.000000,2.000000,1.000000,1.000000,1.000000,1.000000
25%,21.000000,1.000000,3.000000,1.000000,4.000000,4.000000,3.000000,4.000000,4.000000,4.000000,...,3.000000,2.000000,3.250000,2.000000,3.250000,4.000000,3.000000,4.000000,3.000000,4.000000
50%,21.500000,1.000000,4.000000,2.000000,4.000000,4.000000,4.000000,4.000000,4.500000,4.000000,...,4.000000,2.000000,4.000000,2.000000,4.000000,4.000000,4.000000,4.000000,4.000000,4.000000
75%,23.000000,2.000000,5.000000,5.000000,4.000000,4.000000,4.000000,5.000000,5.000000,5.000000,...,4.000000,3.000000,4.000000,3.000000,4.000000,4.000000,4.000000,4.000000,4.000000,4.000000
max,37.000000,3.000000,5.000000,5.000000,5.000000,5.000000,5.000000,5.000000,5.000000,5.000000,...,5.000000,5.000000,5.000000,5.000000,5.000000,5.000000,5.000000,5.000000,5.000000,5.000000


In [ ]:
!pip install pingouin

   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 204.8/204.8 kB 10.4 MB/s eta 0:00:00


In [ ]:
PP = df[['Performance Pressure_1', 'Performance Pressure_2', 'Performance Pressure_3']]
CE = df[['Competency_1', 'Competency_2', 'Competency_3']]

In [ ]:
import pingouin as pg

pg.cronbach_alpha(PP)
pg.cronbach_alpha(CE)

(np.float64(0.6895957913859663), array([0.574, 0.778]))

**Cronbach’s alpha for the Performance Pressure (PP) scale was 0.69, indicating marginal internal consistency. While close to the commonly accepted threshold of 0.70, the relatively wide confidence interval suggests variability in item coherence.**

In [ ]:
PP12 = df[['Performance Pressure_1', 'Performance Pressure_2']]
CE12 = df[['Competency_1', 'Competency_2']]

pg.cronbach_alpha(PP12)
pg.cronbach_alpha(CE12)

(np.float64(0.7119997561644671), array([0.58 , 0.803]))

**The Performance Pressure scale demonstrated marginal internal consistency (α = 0.69). Item-level analysis indicated that the third item exhibited weaker alignment with the other indicators, suggesting some variation in how different dimensions of performance expectations are perceived.**

DISCUSSION:
- AI may increase different types of pressure, not one single dimension (PP)

**The Competency Escalation scale showed marginal internal consistency, with item-level analysis suggesting that the item related to continuous skill updating may capture a distinct, more long-term dimension of perceived expectations.**

DISCUSSION:
- Students distinguish between needing better skills now vs. constantly updating skills over time (CE)



In [ ]:
PP13 = df[['Performance Pressure_1', 'Performance Pressure_3']]
CE13 = df[['Competency_1', 'Competency_3']]


pg.cronbach_alpha(PP13)
pg.cronbach_alpha(CE13)

(np.float64(0.46052933351681125), array([0.213, 0.63 ]))

In [ ]:
PP23 = df[['Performance Pressure_2', 'Performance Pressure_3']]
CE23 = df[['Competency_2', 'Competency_3']]


pg.cronbach_alpha(PP23)
pg.cronbach_alpha(CE23)

(np.float64(0.5670549581839905), array([0.369, 0.703]))

In [ ]:
# Composite Variables
df['PP_score'] = PP.mean(axis=1)
df['CE_score'] = CE.mean(axis=1)

In [ ]:
df[['PP_score', 'CE_score']].describe()

,PP_score,CE_score
count,110.000000,110.000000
mean,3.854545,3.866667
std,0.745393,0.699789
min,1.666667,1.666667
25%,3.666667,3.666667
50%,4.000000,4.000000
75%,4.333333,4.333333
max,5.000000,5.000000


PP mean = 3.85

CE mean = 3.87


Students perceive both performance pressure and competency escalation at equally high levels

PP std = 0.75

CE std = 0.70

People are responding consistently across both constructs.

In [ ]:
from scipy.stats import ttest_rel

ttest_rel(df['PP_score'], df['CE_score'])

TtestResult(statistic=np.float64(-0.1856806534798496), pvalue=np.float64(0.8530402954470684), df=np.int64(109))

In [ ]:
df[['PP_score', 'CE_score']].corr()

,PP_score,CE_score
PP_score,1.00000,0.55266
CE_score,0.55266,1.00000


Correlation (r = 0.55)

Moderate positive correlation.

Meaning:
- When someone perceives more performance pressure
They ALSO tend to perceive more competency escalation

But, not super high (not like 0.8+).

So, they are related, but NOT the same thing.

Core conceptual finding:
AI is not just increasing one type of expectation, but multiple linked but distinct pressures.

**Findings**

Both performance pressure (M = 3.85) and competency escalation (M = 3.87) were perceived at similarly high levels among respondents.

The two constructs were moderately correlated (r = 0.55), suggesting that while related, they represent distinct dimensions of perceived workplace expectations.

AI makes people feel like they must:
- work harder/faster (PP)
- be more skilled (CE)

And those pressures:
- move together
- but are not identical


In [ ]:
IV5 = df[['Perceptions_1','Perceptions_2','Perceptions_3','Perceptions_4','Perceptions_5']]
df['IV5_score'] = IV5.mean(axis=1)

pg.cronbach_alpha(IV5)

(np.float64(0.5709304051046149), array([0.43 , 0.686]))

Initial reliability analysis indicated low internal consistency (α = 0.57) when combining all perception items. Items reflecting personal confidence were conceptually distinct from perceived AI integration and were therefore excluded. The final composite variable focused on perceived AI prevalence and use in the workplace.

## Construct Refinement

In [ ]:
IV345 = df[['Perceptions_3','Perceptions_4','Perceptions_5']]
df['IV345_score'] = IV345.mean(axis=1)

pg.cronbach_alpha(IV345)

(np.float64(0.5959358808290156), array([0.445, 0.711]))

In [ ]:
# REFINE IV - AI Prevalence / Adoption - Perceived AI presence / integration in the workplace

IV = df[['Perceptions_4','Perceptions_5']]
df['IV_score'] = IV.mean(axis=1)

pg.cronbach_alpha(IV)

(np.float64(0.7276776246023335), array([0.603, 0.813]))

Perceived AI integration was operationalized using two items reflecting the increasing prevalence and future dominance of AI in the workplace. These items demonstrated acceptable internal consistency (α = 0.73).

Normally:
“2-item scales are weaker”

BUT:
- alpha = 0.73 (solid)
- items are conceptually tight
- study is not psychometrics-heavy

So, this is absolutely defensible

**Perceived AI integration was operationalized using two items reflecting the increasing prevalence and future dominance of AI in the workplace. These items demonstrated acceptable internal consistency (α = 0.73).**

DEFINITION
Perceived AI integration was operationalized as the extent to which respondents believe AI is understood, prevalent, and actively used in workplace contexts.

Measuring... how much students perceive AI as present, important, and influential in the workplace

IV → perceived AI integration

DV1 → performance pressure

DV2 → competency escalation

In [ ]:
IV = df[['Perceptions_4', 'Perceptions_5']]
df['IV_score'] = IV.mean(axis=1)

In [ ]:
df[['IV_score', 'PP_score', 'CE_score']].corr()

,IV_score,PP_score,CE_score
IV_score,1.000000,0.463022,0.278749
PP_score,0.463022,1.000000,0.552660
CE_score,0.278749,0.552660,1.000000


Initial reliability analysis indicated that the full 5-item scale lacked internal consistency (α = .57). After refinement, a 2-item scale capturing perceived AI integration demonstrated acceptable reliability (α = .73) and was retained for analysis.

Core Finding:
AI is not just increasing one type of expectation, but multiple linked but distinct pressures.

Coorelation Insight:
PP ↔ CE = 0.55 → moderate
IV ↔ PP = 0.46 → solid
IV ↔ CE = 0.28 → weaker

Key pattern:
	•	AI perception → stronger effect on performance pressure (0.46)
	•	AI perception → weaker effect on competency escalation (0.28)

⸻


Students are thinking:

“AI means I have to work harder/faster RIGHT NOW”


More than:

“AI means I need to become more skilled LONG-TERM”


**Perceived AI integration was positively associated with both performance pressure and competency escalation. However, the relationship was notably stronger for performance pressure (r = .46) than for competency escalation (r = .28), suggesting that students interpret AI primarily as increasing immediate productivity demands rather than long-term skill requirements.**

**While the two constructs were moderately correlated (r = .55), indicating they co-occur, the difference in their relationship with AI perception suggests that they represent distinct dimensions of perceived workplace expectations.**

Work intensification → immediate pressure

Employability → long-term skills


And data shows:
students feel the first more strongly than the second

This:
PP mean ≈ 3.85
CE mean ≈ 3.87

Combined with:
low variance + consistent responses

Means:
Students broadly AGREE AI raises expectations

Not divided. Not controversial.

That’s a consensus perception, which strengthens argument.


In [ ]:
import statsmodels.api as sm

# Independent variable
X = df['IV_score']
X = sm.add_constant(X)

# Model 1: Performance Pressure
model_PP = sm.OLS(df['PP_score'], X).fit()

# Model 2: Competency Escalation
model_CE = sm.OLS(df['CE_score'], X).fit()

print(model_PP.summary())
print(model_CE.summary())

                            OLS Regression Results                            
Dep. Variable:               PP_score   R-squared:                       0.214
Model:                            OLS   Adj. R-squared:                  0.207
Method:                 Least Squares   F-statistic:                     29.47
Date:                Thu, 19 Mar 2026   Prob (F-statistic):           3.52e-07
Time:                        07:35:54   Log-Likelihood:                -109.99
No. Observations:                 110   AIC:                             224.0
Df Residuals:                     108   BIC:                             229.4
Df Model:                           1                                         
Covariance Type:            nonrobust                                         
                 coef    std err          t      P>|t|      [0.025      0.975]
------------------------------------------------------------------------------
const          1.4779      0.442      3.341      0.0

FINDINGS
Perceived AI integration was positively associated with both performance pressure (β = 0.55, p < .001) and competency escalation (β = 0.31, p = .003). However, the relationship was substantially stronger for performance pressure, indicating that students interpret AI primarily as increasing immediate productivity demands rather than long-term skill requirements. The model explained 21.4% of the variance in performance pressure, compared to 7.8% for competency escalation.

⸻

Interpretation

These findings suggest that students perceive AI not only as a driver of increased expectations, but specifically as intensifying short-term performance demands more than long-term skill development. This aligns more closely with work intensification perspectives than with traditional human capital or employability frameworks.